### DISCUSSION TITLE
blog/new-bing-chat/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#322] [Announcements]
# blog/new-bing-chat/

昨天我的新必应体验申请通过了，立马去体验了下，本文记录下我对新必应的最初印象。

https://jimmysong.io/blog/new-bing-chat/

**Comment by qiaohaoforever** [comment]
X-Forwarded-For =8.8.8.8 合适

**Reply by rootsongjc** [reply]
现在这种方式好像用不了，还是被必应检测到，要求下载 Edge 浏览器体验。
